CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
Just realized that we didn't update the CHANGELOG since 8.0... I'll update it now in this branch so you don't need to rebase/merge.